doc/releases/changelog-dev.md

```diff
@@ -258,6 +258,10 @@
 
 <h3>Bug fixes 🐛</h3>
 
+* `_try_wrap_in_custom_ctrl_op` in `pennylane/ops/op_math/controlled.py` now directly generates the `ControlledOp`,
+  bypassing `ControlledQubitUnitary`, to avoid a known bug [(#1494)](https://github.com/PennyLaneAI/catalyst/issues/1494) found in Catalyst.
```

Comment:
```suggestion
  bypassing `ControlledQubitUnitary`, to avoid a [known bug](https://github.com/PennyLaneAI/catalyst/issues/1494) found in Catalyst.
```